puppet-foreman_scap_client/puppet-foreman_scap_client.spec

```diff
@@ -0,0 +1,44 @@
+%global puppet_module foreman_scap_client
+%global puppet_full_name theforeman-%{puppet_module}
+
+%global puppet_modules_dir %{_datadir}/puppet/modules
+
+Name:       puppet-%{puppet_module}
+Version:    0.3.11
+Release:    1%{?dist}
+Summary:    Puppet module to configure foreman_scap_client
+License:    GPLv2
+URL:        https://github.com/theforeman/%{name}
+Source0:    https://forgeapi.puppetlabs.com/v3/files/%{puppet_full_name}-%{version}.tar.gz
+BuildArch:  noarch
+Requires:   puppet >= 2.7.0
+Requires:   puppetlabs-stdlib >= 4.2.0
+
+%description
+Foreman SCAP client Puppet Module configures the client of the same name
+to run scans and upload results to foreman proxy.
+
+%prep
+%setup -qn %{puppet_full_name}-%{version}
+
+%build
+
+%install
+mkdir -p %{buildroot}/%{puppet_modules_dir}/%{puppet_module}
+cp -rp %{puppet_modules_dir}/%{puppet_module} %{buildroot}/%{puppet_modules_dir}/%{puppet_module}/
```

Comment:
This is failing, check the source exists (probably `cp -rp . %{buildroot}/%{puppet_modules_dir}/%{puppet_module}` or similar), you need to be copying from the unpacked source, not the installation dir.